lite/examples/object_detection/ios/ObjectDetection/Extensions/CVPixelBufferExtension.swift

```diff
@@ -26,8 +26,9 @@ extension CVPixelBuffer {
     let imageHeight = CVPixelBufferGetHeight(self)
 
     let pixelBufferType = CVPixelBufferGetPixelFormatType(self)
-
-    assert(pixelBufferType == kCVPixelFormatType_32BGRA)
+ 
```

Comment:
There seems to be still a single space on this line. Please remove it, because otherwise our internal CI wouldn't allow merging this change.